netsim-core/Cargo.toml

```diff
@@ -9,3 +9,6 @@ license = "Apache-2.0"
 [dependencies]
 anyhow = "1.0.79"
 logos = "0.14.0"
+
+[features]
+thread_sleep_until = []
```

Comment:
This fixes a clippy warning about a `#[cfg(feature = "thread_sleep_until")]` check. There's an unstable rust feature named `thread_sleep_until`, but I don't think you can check for unstable features (or for a nightly rustc version) without adding a build script, and this was easier.

Reply:
that's new. Clippy is quite annoying. I thought adding the `#[cfg()]` check would be future proofing the build/lint and leave a hint of what would make more sense to use in the future once stabilised.

Thanks for adding this and fixing the lint